uwblueprint/sistering
# Modal for removing volunteers

## Ticket link

Closes #159



## Implementation description
* Created a modal for removing volunteers.



## Steps to test
1. Open the app
2. Go to Remove Volunteer Demo in menu bar
3. Enter a name of a volunteer you would like to remove
4. Verify that the modal works and looks as expected



## What should reviewers focus on?
* Styling of the modal matches mockup


## Checklist
